src/Panel.js

```diff
@@ -23,6 +25,19 @@ const Panel = React.createClass({
     };
   },
 
+  getInitialState(){
+    let defaultExpanded = this.props.defaultExpanded != null ?
+      this.props.defaultExpanded :
+        this.props.expanded != null ?
+        this.props.expanded :
+        false;
+
+    return {
+      expanded: defaultExpanded,
+      collapsing: false
```

Comment:
No. Just remove it.

It is used by `CollapsibleMixin` and you are removing this mixin,
then this property becomes obsolete :wink: